specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,289 @@
+  /// representing multiple data types. The browsing data kinds that are supported 
+  /// are listed below. These data kinds follow a hierarchical structure in which 
+  /// nested bullet points are included in their parent bullet point's data kind.
+  /// Ex: DOM storage is encompassed in site data which is encompassed in the profile data. 
+  /// * Profile
+  ///   * Site Data
+  ///     * DOM Storage: App Cache, File Systems, Indexed DB, Local Storage, Web SQL, Cache 
+  ///         Storage
+  ///     * Cookies 
+  ///   * HTTP Cache 
+  ///   * Download History
+  ///   * General Autofill 
+  ///   * Password Autosave
+  ///   * Browsing History
+  ///   * Settings  
+  /// The completed handler will be invoked when the browsing data has been cleared and will 
+  /// indicate if the specified data was properly cleared.
+  /// If the WebView object is closed before the clear browsing data operation 
+  /// has completed, the handler will be released, but not invoked. In this case 
+  /// the clear browsing data operation may or may not be completed. 
+  /// ClearBrowsingData clears the `dataKinds` for all time. 
+  HRESULT ClearBrowsingData(
+      [in] UINT64 dataKinds,
+      [in] ICoreWebView2ClearBrowsingDataCompletedHandler* handler);
+  
+  /// ClearBrowsingDataInTimeRange takes in two additional parameters for the 
+  /// start and end time for which it should clear the data between.  The startTime and endTime 
+  /// parameters correspond to the number of seconds since the UNIX epoch. 
+  ///
+  /// \snippet AppWindow.cpp ClearBrowsingData
+  HRESULT ClearBrowsingDataInTimeRange(
+      [in] UINT64 dataKinds, 
+      [in] double startTime,
+      [in] double endTime, 
+      [in] ICoreWebView2ClearBrowsingDataCompletedHandler* handler);
+}
+
+/// The caller implements this interface to receive the ClearBrowsingData result.
+[uuid(27676699-FE17-4E2B-8C1B-267395A04ED5), object, pointer_default(unique)]
+interface ICoreWebView2ClearBrowsingDataCompletedHandler : IUnknown {
+  /// Provide the completion status and result of the corresponding
+  /// asynchronous method. The result indicates if the ClearBrowsingData 
+  /// call has completed successfully and all intended data has been cleared. 
+  HRESULT Invoke(
+      [in] HRESULT errorCode, 
+      [in] BOOL isSuccessful);
+}
+
+```
+### .NET, WinRT
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+    [Flags] enum CoreWebView2BrowsingDataKind
+    {
+        AppCache = 1,
+        FileSystems = 2,
+        IndexedDb = 4,
+        LocalStorage = 8,
+        WebSql = 16,
+        CacheStorage = 32,
+        DomStorage = 63,
+        Cookies = 64,
+        Site = 127,
+        HttpCache = 128,
+        DownloadHistory = 256,
+        GeneralAutofill = 512,
+        PasswordAutosave = 1024,
+        BrowsingHistory = 2048,
+        Settings = 4096,
+        Profile = 8191,
+    };
+
+    public partial class CoreWebView2Environment
```

Comment:
This still says environment. If you run build-apiwriter you can open the generated Microsoft.Web.WebView2.Core.idl file and find the generated IDL to paste in here.